common/src/test/java/org/jboss/pnc/common/json/ModuleConfigJsonTest.java

```diff
@@ -0,0 +1,60 @@
+package org.jboss.pnc.common.json;
+
+import java.io.IOException;
+import java.net.URL;
+
+import org.jboss.pnc.common.json.moduleconfig.JenkinsBuildDriverModuleConfig;
+import org.jboss.pnc.common.json.moduleconfig.MavenRepoDriverModuleConfig;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class ModuleConfigJsonTest {
+
+    @Test
+    public void serialize() {
```

Comment:
We may change the name to something a little bit better:
* testSerializationWithSuccess
* shouldSucceedOnSerializingProperConfiguration etc